### PR TITLE
Add one subclass to admin

### PR DESCRIPTION
**Pull request by core23**
Would be great if you can add one simple subclass to an admin instead of replacing the whole subclasses by using ``setSubClasses``.

### CODE REVIEW

**Comment by OskarStark** [comment]
This is a BC break

**Comment by OskarStark** [comment]
What about adding the method and create a separate PR for 3.0 where we add this method to the Interface ?

**Comment by Soullivaneuh** [comment]
> What about adding the method and create a separate PR for 3.0 where we add this method to the Interface ?

:+1:

**Comment by core23** [comment]
@Soullivaneuh Remove the interface. After merging this, I will provide another PR for the interface to keep this in mind.

**Comment by OskarStark** [comment]
Do we have to check if the subclass already exists in this array?

**Comment by core23** [comment]
Thanks for your advice @OskarStark 
Added a check

**Comment by Soullivaneuh** [comment]
ping @rande for opinion.

File: Admin/Admin.php [hunk 1]
```diff
@@ -1078,6 +1078,16 @@ public function getSubClasses()
     /**
      * {@inheritdoc}
```
Comment (OskarStark) [inline Admin/Admin.php]: can we use inheritdoc here?

**Comment by core23** [comment]
@OskarStark There is no more bc break. So you could merge this in the current branch.

**Comment by OskarStark** [comment]
Thank you @core23

**Comment by core23** [comment]
@OskarStark Same here. You could remove the milestone flag

**Comment by OskarStark** [comment]
:+1:

**Pull request merged**